src/components/molecules/form/form.svelte

```diff
@@ -1,5 +1,5 @@
 <script>
-  import Button from '../button.svelte';
+  import Button from '../../atoms/button/button.svelte';
```

Comment:
```suggestion
  import Button from '~/components/atoms/button/button.svelte';
```

we may need to setup ~ alias first

Reply:
can't use ~ this returns error Module not found.